site/content/docs/developing/environment-variables.en.md

```diff
@@ -56,6 +56,22 @@ Here's a quick guide showing you how to add environment variables to your app by
 
 ![Editing the manifest to add env vars](https://raw.githubusercontent.com/kohidave/ecs-cliv2-demos/master/env-vars-edit.svg?sanitize=true)
 
+Additionally, if you want to specify environment variables in bulk, you can add them in your [env file](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) and pass its location to your [manifest](../manifest/overview.en.md) in the `env_file` section.
```

Comment:
```suggestion
Additionally, if you want to add environment variables in bulk, you can list them in an [env file](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) and specify its location in the `env_file` field of your [manifest](../manifest/overview.en.md).
```

Reply:
Can we also mention that the path should be from the root of the workspace?